src/LinuxTracing/LibunwindstackUnwinder.h

```diff
@@ -24,6 +24,24 @@
 
 namespace orbit_linux_tracing {
 
+// This class is a "view" of a stack slice (some copy of process memory). It contains a bare pointer
+// to the data field of actual stack slice.
```

Comment:
"to the data field of actual stack slice" sounds a bit broken. Maybe "It contains a bare pointer to the array that actually owns the stack data.".